src/DotLiquid/StandardFilters.cs

```diff
@@ -227,6 +227,26 @@ public static string StripHtml(string input)
         }
 
         /// <summary>
+        /// Converts the input object into a formatted currency as specified by the culture info.
+        /// </summary>
+        /// <param name="input"></param>
+        /// <param name="cultureInfo"></param>
+        /// <returns></returns>
+        public static string Currency(object input, string cultureInfo = "en-US")
+        {
+            decimal amount;
+
+            if (decimal.TryParse(input.ToString(), out amount))
+            {
+                var culture = new CultureInfo(cultureInfo);

```

Comment:
It is effectively best to take the current culture as default, thanks.
It doesn't fix the user override issue but since the specific constructor is not part of .net standard 1.3, that should do it.